src/components/learner-credit-management/tests/BudgetCard.test.jsx

```diff
@@ -62,164 +63,181 @@ const BudgetCardWrapper = ({ ...rest }) => (
 );
 
 describe('<BudgetCard />', () => {
-  describe('with enterprise offer', () => {
-    beforeEach(() => {
-      jest.clearAllMocks();
-    });
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
 
-    it('displays correctly for Offers', () => {
-      const mockOffer = {
-        id: mockEnterpriseOfferId,
-        name: mockOfferDisplayName,
-        start: '2022-01-01',
-        end: '2023-01-01',
-      };
-      const mockOfferRedemption = {
-        created: '2022-02-01',
-        enterpriseEnrollmentId: mockEnterpriseOfferEnrollmentId,
-      };
-      useOfferSummary.mockReturnValue({
-        isLoading: false,
-        offerSummary: {
-          totalFunds: 5000,
-          redeemedFunds: 200,
-          remainingFunds: 4800,
-          percentUtilized: 0.04,
-          offerType: 'Site',
-          budgetsSummary: [
-            {
-              id: 123,
-              start: '2022-01-01',
-              end: '2022-01-01',
-              available: 200,
-              spent: 100,
-              enterpriseSlug: enterpriseId,
-            },
-          ],
-        },
-      });
-      useOfferRedemptions.mockReturnValue({
-        isLoading: false,
-        offerRedemptions: {
-          results: [mockOfferRedemption],
-          itemCount: 1,
-          pageCount: 1,
-        },
-        fetchOfferRedemptions: jest.fn(),
-      });
-      render(<BudgetCardWrapper
-        offer={mockOffer}
-        enterpriseUUID={enterpriseUUID}
-        enterpriseSlug={enterpriseId}
-      />);
-      expect(screen.getByText('Overview'));
-      expect(screen.queryByText('Executive Education')).not.toBeInTheDocument();
-      const formattedString = `Expired ${dayjs(mockOffer.end).format('MMMM D, YYYY')}`;
-      const elementsWithTestId = screen.getAllByTestId('offer-date');
-      const firstElementWithTestId = elementsWithTestId[0];
-      expect(firstElementWithTestId).toHaveTextContent(formattedString);
+  it('displays correctly for Enterprise Offers', () => {
```

Comment:
[nit] similar to other comments to append enterprise offers with the api call origin